resources/sass/app.scss

```diff
@@ -381,6 +381,10 @@ dl.import-meta .download-csv {
     }
 }
 
+.heading-button, .random-mismatches {
+    flex-shrink: 0; // don't show ellipsis as long as the text fits
+}
+
```

Comment:
This didn't work as long as it was inside the `.cdx-button` selector, i'm not sure why. I extracted it and it works like this.

Reply:
ahhhh, this is what you mean.... i added a comment suggesting to do this.. will check again and get back to you